bootstrap/Makefile.bootstrap

```diff
@@ -0,0 +1,24 @@
+export SHELL = /bin/bash
+export BUILD_HARNESS_ORG ?= plus3it
+export BUILD_HARNESS_PROJECT ?= tardigrade-ci
+export BUILD_HARNESS_BRANCH ?= master
+export BUILD_HARNESS_PATH ?= $(shell until [ -d "$(BUILD_HARNESS_PROJECT)" ] || [ "`pwd`" == '/' ]; do cd ..; done; pwd)/$(BUILD_HARNESS_PROJECT)
+
+MAKEFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
```

Comment:
kinda curious about these three being here, since it seems they're actually used in `bootstrap/Makefile`?

Reply:
having them in the bootstrap/Makefile was a mistake, updated.

Reply:
heh, that's kind of the opposite of what i meant with the question... was wondering why they are set in _this_ file at all?

Reply:
lol woosh - so the `bin/install.sh` downloads the `Makefile.bootstrap` file into the root of the consumer project's directory. I need that directory context passed back up to `tardigrade-ci/bootstrap/Makefile` so it knows 1) where to find the `Dockerfile` and 2) what to call the resulting image from the `docker build` call

Reply:
got it. and for some reason these do not need exporting?

Reply:
those export statements are copy pasta from the cloudposse buildharness. My guess is that they're intended for use in other shell scripts but not 100% on that